### PR TITLE
Propose PHP <= 5.5

### PR DESCRIPTION
**Pull request by infabo**
Propose PHP <= 5.5 for 1.x branch.

### CODE REVIEW

**Comment by lisachenko** [comment]
Hi, there are errors in the composer.lock, please resolve them.

File: composer.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
     "license": "MIT",
 
     "require": {
-        "php": ">=5.5.0",
+        "php": "<=5.5.99",
```
Comment (lisachenko) [inline composer.json]: It's better to use ~5.5.0 here.

File: composer.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
         "jakubledl/dissect": "~1.0",
-        "doctrine/annotations": "~1.0",
+        "doctrine/annotations": "<1.3.0",
```
Comment (lisachenko) [inline composer.json]: Please, switch to ~1.2.0

**Review by lisachenko — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (infabo) [reply composer.json]: This would restrict the 1.x branch to PHP 5.5 only. Any reasons why not support PHP 5.3 / 5.4?

File: composer.json (hunk 2 above)
Comment (infabo) [reply composer.json]: acknowledged.

**Comment by infabo** [comment]
Can't find any issues in composer.lock.

File: composer.json (hunk 1 above)
Comment (lisachenko) [reply composer.json]: 5.3 and 5.4 are not supported, anyone who wants to use an old version, can switch to 0.x versions.

**Comment by lisachenko** [comment]
Merged, thanks

**Comment by lisachenko** [comment]
Also, removed the `composer.lock` file from the repository.

**Pull request merged**